Runtime/org.codelite.codelite.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.codelite.codelite</id>
+  
+  <name>CodeLite</name>
+  <summary>CodeLite is an open source IDE for the C//C++ languages</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  
+  <description>
+    <p>
+      CodeLite is an open source IDE for the C//C++ languages
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">codelite.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://screenshots.debian.net/shrine/screenshot/simage/large-e7feb090ac27fb2959d53b522e07c84e.jpg</image>
+    </screenshot>
+    <screenshot>
+      <image>https://screenshots.debian.net/shrine/screenshot/simage/large-5ba5845aec9fa7a591d7ab0f5e6a9605.jpg</image>
```

Comment:
Wonder if diff-view and database query are the best/more representative screenshots of the application ;-)

Reply:
I could upload to some new screenshots to https://codelite.org and we could use them instead

Reply:
Yes, to be honest I just package this in Solus so I think it is better as you say to add new better pictures, just ping me here and I will use them instead.